boost-maven/boost-maven-plugin/src/it/test-mpHealth-1.0/src/test/java/it/io/openliberty/guides/health/HealthTestIT.java

```diff
@@ -0,0 +1,69 @@
+// tag::comment[]
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - Initial implementation
+ *******************************************************************************/
+// end::comment[]
+// tag::HealthTest[]
+package it.io.openliberty.guides.health;
+
+import static org.junit.Assert.assertEquals;
+import java.util.HashMap;
+import javax.json.JsonArray;
+import org.junit.After;
+import org.junit.Test;
+
+public class HealthTestIT {
+
+    private JsonArray servicesStates;
+    private static HashMap<String, String> dataWhenServicesUP;
+    private static HashMap<String, String> dataWhenInventoryDown;
+
+    static {
```

Comment:
Should this method just be an init() or setup() with the `@BeforeClass` annotation?